docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Why are we only showing the first 2 rows? What is the point?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>